pkg/reconciler/instances/ory/action.go

```diff
@@ -21,10 +21,11 @@ import (
 )
 
 const (
-	oryChart     = "ory"
-	oryNamespace = "kyma-system"
-	jwksAlg      = "RSA256"
-	jwksBits     = 2048
+	oryChart        = "ory"
+	oryNamespace    = "kyma-system"
```

Comment:
Do we really need this const? Ory reconciliation should take place in namespace in which it is said to. From `Context` API perspective it can be retrieved in the following way: `context.Task.Namespace`

Reply:
This would require more refactoring, as it is used in most of those actions